### PR TITLE
refactor: unify bindings and extensions

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:

Unify bindings and extensions:
- as previous PRs, styles will be done in https://github.com/asyncapi/shape-up-process/issues/86
- render bindings and extensions as schema,
- render missed JSON-Schema (we don't also render them in HTML-template) like: `if/then/else`, `contains`, `propertyNames`, `patternProperties`, `not`. 
 
Examples:
image

Not:
image

Contains:
image

PropertyNames:
image
    
**Related issue(s)**
Part of https://github.com/asyncapi/shape-up-process/issues/87

### CODE REVIEW

**Comment by derberg** [comment]
@aeworxet would you like to do a review? I would approve once you approve so Maciek can merge

**Comment by aeworxet** [comment]
LGTM. :)

**Review by derberg — APPROVED** [review]

**Comment by magicmatatjahu** [comment]
@aeworxet @derberg Thanks for review guys! :)

**Pull request merged**
